python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -40,6 +40,7 @@ from cuml.common.handle cimport cumlHandle
 from cuml.ensemble.randomforest_shared cimport *
 from cuml.utils import get_cudf_column_ptr, get_dev_array_ptr, \
     input_to_dev_array, zeros
+from cuml.utils.cupy_utils import rmm_cupy_ary
```

Comment:
Minor nitpick, but this import seems to be inconsistent with the way our other imports are done (using `cuml.utils` rather than `cuml.utils.util_class`). We should strive for consistency.